ui/nuxeo-document-preview.js

```diff
@@ -211,6 +220,14 @@ import './viewers/nuxeo-video-viewer.js';
 
       // lookup the preview according to the blob's mimetype
       const previewers = dom(this.root).querySelectorAll('template');
+
+      // Use embed preview when available
+      if ('preview' in this._blob) {
```

Comment:
So this would be the same as `if (this._blob.preview)`, right? Or does `in` cover some other use case?